test/Helpers/TestHelper.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace OAuth.Net.Tests
+{
+    public class TestHelper
+    {
+        private readonly string _apiKey;
+        private readonly string _clientSecret;
+        private readonly string _authToken;
+        private readonly string _authTokenSecret;
+
+        public TestHelper(string apiKey, string secret, string authToken, string authTokenSecret)
+        {
+            _apiKey = apiKey;
+            _clientSecret = secret;
+            _authToken = authToken;
+            _authTokenSecret = authTokenSecret;
+        }
+
+        /// <summary>
+        /// Using reflection, it will call the method that computes the oauth signature
```

Comment:
Since you removed the usage of reflection, this comment is no longer true.

Reply:
Fixed.

Reply:
I think you forgot to push the change? I can't see it here in GitHub at least.

Reply:
Was going to wait and push both at the same time :).